acrobat/scripts/scripts.js

```diff
@@ -339,4 +339,9 @@ const { ietf } = getLocale(locales);
       window.dispatchEvent(imsIsReady);
     }
   }, 1000);
+
+  if (getMetadata('commerce') === 'true') {
```

Comment:
this can probably just be:
```
if (getMetadata('commerce') ) {...}
if it's not there it's `false`

Reply:
what if it returns the string `'false'`?

Reply:
> what if it returns the string `'false'`?

LOL then it would be `true` but this function is bool only